LinkedList.java

```diff
@@ -0,0 +1,206 @@
+package linkedListFunctions;
+
+public class LinkedList {
+	Node head;

```

Comment:
Mention the access specifier to keep it clean.

Reply:
by default, it is private, is it fine to be private right?